providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/FlagdOptionsTest.java

```diff
@@ -1,29 +1,24 @@
 package dev.openfeature.contrib.providers.flagd;
 
 import io.opentelemetry.api.OpenTelemetry;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import org.mockito.Mockito;
 
 import java.util.function.Function;
 
-import static dev.openfeature.contrib.providers.flagd.Config.DEFAULT_CACHE;
-import static dev.openfeature.contrib.providers.flagd.Config.DEFAULT_HOST;
-import static dev.openfeature.contrib.providers.flagd.Config.DEFAULT_MAX_CACHE_SIZE;
-import static dev.openfeature.contrib.providers.flagd.Config.DEFAULT_MAX_EVENT_STREAM_RETRIES;
-import static dev.openfeature.contrib.providers.flagd.Config.DEFAULT_PORT;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static dev.openfeature.contrib.providers.flagd.Config.*;
```

Comment:
nit - wildcard imports :)

Reply:
what is the general rule regarding imports, if we do not want to use wildcard imports, we should activate the checkstyle rule, to prevent them